CardLogic.php

```diff
@@ -1334,25 +1334,12 @@ function ProcessTrigger($player, $parameter, $uniqueID, $target="-", $additional
         $index = FindCharacterIndex($player, "HVY648");
         CharacterChooseSubcard($player, $index, isMandatory:false);
         AddDecisionQueue("ADDDISCARD", $player, "-", 1);
-        AddDecisionQueue("MULTIZONEINDICES", $player, "THEIRITEMS:minCost=0;maxCost=1", 1);
+        AddDecisionQueue("MULTotherPlayerIZONEINDICES", $player, "THEIRITEMS:minCost=0;maxCost=1", 1);

```

Comment:
I feel like this otherPlayer here is probably a typo that would freeze the game

Reply:
Definitely a typo! Fixed